tests/csv_upload_tests.py

```diff
@@ -279,23 +280,13 @@ def test_import_csv(setup_csv_upload, create_csv_files):
     # make sure that john and empty string are replaced with None
     engine = get_upload_db().get_sqla_engine()
     data = engine.execute(f"SELECT * from {CSV_UPLOAD_TABLE}").fetchall()
-    if utils.backend() == "hive":
```

Comment:
@bkyryliuk this special casing no longer seems necessary when using the Parquet file format.

Reply:
nice!